build.zig

```diff
@@ -13,6 +13,7 @@ fn addFuzzStep(b: *Build, target: std.Build.ResolvedTarget, afl_clang_lto_path:
     const fuzz_step = b.step("fuzz", "Build executable for fuzz testing.");
     var fuzz_target = target;
     fuzz_target.result.ofmt = .c;
+    fuzz_target.query.ofmt = .c;
```

Comment:
`resolveTargetQuery` does the same thing but if you want to avoid touching internal parts you can use that instead.
```suggestion
    const fuzz_target = blk: {
        var query = target.query;
        query.ofmt = .c;
        break :blk b.resolveTargetQuery(query);
    };
```